### PR TITLE
App based auth additions

### PR DESCRIPTION
**Pull request by pbafna03**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by get-itips — APPROVED** [review]

File: teams/teams-ps/teams/Connect-MicrosoftTeams.md [hunk 1]
```diff
@@ -79,7 +92,45 @@ Account                 Environment 	Tenant                                Tenan
...
+This example demonstrates how to authenticate using a certificate thumbprint. Application-based authentication has been reintroduced in preview with version 4.7.1-preview. For details and supported cmdlets, please see [Application-based authentication in Teams PowerShell Module](/MicrosoftTeams/teams-powershell-application-authentication).
+
+```powershell
+Connect-MicrosoftTeams -CertificateThumbprint "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX" -ApplicationId "00000000-0000-0000-0000-000000000000" -TenantId "YYYYYYYY-YYYY-YYYY-YYYY-YYYYYYYYYYYY"
+```
+
+### Example 5: Connect to MicrosoftTeams using Application-based Access Tokens
+This example demonstrates how to authenticate with an application using Access Tokens. Access Tokens can be retrieved via the login.microsoftonline.com endpoint. It requires two Access Tokens – “MS Graph” and “Skype and Teams Tenant Admin API” resources.
+
+Application-based authentication has been reintroduced in preview with version 4.7.1-preview. For details and supported cmdlets, please see [Application-based authentication in Teams PowerShell Module](/MicrosoftTeams/teams-powershell-application-authentication).
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Connect-MicrosoftTeams.md:105]:
```suggestion
Application-based authentication has been reintroduced in preview with version 4.7.1-preview. For details and supported cmdlets, see [Application-based authentication in Teams PowerShell Module](/MicrosoftTeams/teams-powershell-application-authentication).
```

File: teams/teams-ps/teams/Connect-MicrosoftTeams.md [hunk 2]
```diff
@@ -119,9 +170,11 @@ user@contoso.com        AzureCloud   xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx  xxxxx
 ## PARAMETERS
 
 ### AccessTokens
-Specifies a access tokens for "MSGraph" and "Skype and Teams Tenant Admin API" resources. This new parameter is added in version 2.3.2-preview. 
+Specifies access tokens for "MS Graph" and "Skype and Teams Tenant Admin API" resources. Both the tokens used should be of the same type.
+
+(a) Application-based authentication has been reintroduced in preview with version 4.7.1-preview. For details and supported cmdlets, please see [Application-based authentication in Teams PowerShell Module](/MicrosoftTeams/teams-powershell-application-authentication).
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Connect-MicrosoftTeams.md:175]:
```suggestion
- Application-based authentication has been reintroduced in preview with version 4.7.1-preview. For details and supported cmdlets, see [Application-based authentication in Teams PowerShell Module](/MicrosoftTeams/teams-powershell-application-authentication).
```

File: teams/teams-ps/teams/Connect-MicrosoftTeams.md [hunk 3]
```diff
@@ -119,9 +170,11 @@ user@contoso.com        AzureCloud   xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx  xxxxx
...
 
-Following steps must be performed by Tenant Admin in the Azure portal when using your own application. 
+(b) Delegated flow - The following steps must be performed by Tenant Admin in the Azure portal when using your own application.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Connect-MicrosoftTeams.md:177]:
```suggestion
- Delegated flow - The following steps must be performed by Tenant Admin in the Azure portal when using your own application. 
```

File: teams/teams-ps/teams/Connect-MicrosoftTeams.md [hunk 4]
```diff
@@ -179,10 +232,9 @@ Accept wildcard characters: False
 ```
 
 ### -ApplicationId
-Specifies the application ID of the service principal.
+Specifies the application ID of the service principal that is used in application-based authentication. 
 
-> [!WARNING]
->This parameter has been removed from the latest versions and replaced by the AccessTokens parameter.
+This parameter has been reintroduced in preview with version 4.7.1-preview. For more information about Application-based authentication and supported cmdlets, please see [Application-based authentication in Teams PowerShell Module](/MicrosoftTeams/teams-powershell-application-authentication).
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Connect-MicrosoftTeams.md:237]:
```suggestion
This parameter has been reintroduced in preview with version 4.7.1-preview. For more information about Application-based authentication and supported cmdlets, see [Application-based authentication in Teams PowerShell Module](/MicrosoftTeams/teams-powershell-application-authentication). 
```

File: teams/teams-ps/teams/Connect-MicrosoftTeams.md [hunk 5]
```diff
@@ -195,10 +247,10 @@ Accept pipeline input: False
...
 ```
 
-### -CertificateThumbprint (Removed from version 2.4.1-preview)
-Specifies the certificate thumbprint of a digital public key X.509 certificate of a user account that has permission to perform this action.
-> [!WARNING]
->This parameter has been removed from version 2.4.1-preview.
+### -CertificateThumbprint
+Specifies the certificate thumbprint of a digital public key X.509 certificate of an application that has permission to perform this action.
+
+This parameter has been reintroduced in preview with version 4.7.1-preview. For more information about Application-based authentication and supported cmdlets, please see [Application-based authentication in Teams PowerShell Module](/MicrosoftTeams/teams-powershell-application-authentication).
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Connect-MicrosoftTeams.md:253]:
```suggestion
This parameter has been reintroduced in preview with version 4.7.1-preview. For more information about Application-based authentication and supported cmdlets, see [Application-based authentication in Teams PowerShell Module](/MicrosoftTeams/teams-powershell-application-authentication).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @pbafna03, kindly commit the few minor suggested changes. Many thanks.

**Comment by pbafna03** [comment]
> Copy review - @pbafna03, kindly commit the few minor suggested changes. Many thanks.

@JohanFreelancer9 - done, thanks!

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Pull request merged**